src/parser/parse.ts

```diff
@@ -49,7 +49,7 @@ function parseFileHeader(container: CFB$Container): HWPHeader {
     throw new Error(`FileHeader must be ${FILE_HEADER_BYTES} bytes, Received: ${content.length}`)
   }
 
-  // TODO: (@hahnlee) 파일 시그니처 검증하기
+  const signature = String.fromCharCode(...Array.from(content.slice(0, 17)))
```

Comment:
If the signature doesn't match, it would be better to throw an error.

Reply:
ok. I agree. :) 
I add SIGNATURE const and if throw statement in the second commit